Dockerfile.arm64

```diff
@@ -35,6 +35,8 @@ LABEL name="Calico CLI tool" \
 
 COPY --from=ubi /licenses /licenses
 ADD bin/calicoctl-linux-arm64 /calicoctl
+# Overwrite qemu binary
+ADD LICENSE /usr/bin/qemu-aarch64-static
```

Comment:
At this point our `arm64` image doesn't have executables to invoke normal commands for removing `qemu-aarch64-static` binary, so I just overwrite the file with a text file.

Reply:
How exactly does that help?  Who is it that checks for the existence of /usr/bin/qemu-aarch64-static and then tries to run it?  With this solution, will they still try to run it, and then hit an error because it's just plain text?